src/ResourceManager/Sql/Commands.Sql/Database Backup/Services/AzureSqlDatabaseBackupCommunicator.cs

```diff
@@ -48,6 +48,11 @@ public class AzureSqlDatabaseBackupCommunicator
         /// </summary>
         private static ResourceManagementClient ResourcesClient { get; set; }
 
+        /// <summary>
+        /// The Sql client to be used by this end points communicator
+        /// </summary>
+        private static Management.Sql.SqlManagementClient SqlArmClient { get; set; }
```

Comment:
This is already in SqlClient above - why do you need both?

Reply:
some code is using legacy APIs so needed both